modules/openxr/extensions/openxr_hand_interaction_extension.cpp

```diff
@@ -64,34 +64,34 @@ bool OpenXRHandInteractionExtension::is_available() {
 }
 
 void OpenXRHandInteractionExtension::on_register_metadata() {
-	OpenXRInteractionProfileMetadata *metadata = OpenXRInteractionProfileMetadata::get_singleton();
-	ERR_FAIL_NULL(metadata);
+	OpenXRInteractionProfileMetadata *openxr_metadata = OpenXRInteractionProfileMetadata::get_singleton();
```

Comment:
I had to do a bunch of these changes as it seems `metadata` is a class member of `Object` (or introduced by our ClassDB macro). Bit of a PITA.